console/bridge.go

```diff
@@ -147,9 +148,15 @@ func (b *bridge) OpenWallet(call otto.FunctionCall) (response otto.Value) {
 		if input, err := b.prompter.PromptPassword("Please enter current PUK: "); err != nil {
 			throwJSException(err.Error())
 		} else {
+			if !regexp.MustCompile(`^[0-9]{12,}$`).MatchString(input) {
+				throwJSException("PUK needs to be at least 12 digit long")
+			}
```

Comment:
These seem like the wrong place to handle the errors. The smart card library should ideally tell us what the error is, not the wrapping console.

Reply:
The smartcard library is meant to communicate with several smartcard applications and can not assume what the app will accept or not. This is either to be handled by the client code or the app. We know that the app has a bug and if I'm not mistaken you have already opened an issue in their repo, so in the mean time I think it's safer to have it here.

Reply:
I kind of meant that our `scwallet` package could return these errors. Scope wise putting it in there or here is the same (neither has a clue what the actual JavaCard app wants), but putting it inside `scwallet` would allos Clef to immediately support the same limits, whereas currently we'd need to duplicate this exact same length checks in Clef too.